### ISSUE TITLE
Is this just a Frustration for Linux users from Microsoft?

### ISSUE THREAD

**Issue by markosjal** [Microsoft/vscode#72668]
🚨 The issue tracker is not for questions 🚨

If you have a question, please ask it on https://stackoverflow.com/questions/tagged/visual-studio-code.

**Comment by markosjal** [comment]
Ok I came to see where I could post issues with Visual Studio Code they say I can ask questions here but apparently github says differently Yet anotrher frustration with this product!

I honestly wonder if this is released as a snake in the grass by Microsoft at least the Linux version. Such strange things happen it is IMPOSSIBLE to use I selet text to copy and get different text pasted, sometimes text I never even selected. It is as if it is undermining everything I do. As I worked with it again all day today I kept having to undo what I had done because very strange things were pasted where I did not paste them!


Is this just Microsoft's way of saying 'get WIndows'?

I will go back to nano , I do not need this it is not that helpful for PHP and has done nothing for javascript.

I also want to mention the Black background makes things very hard to read under some conditions, and causes eye strain. Not saying it should be white either, something more neutral would be best .

good luck with the effort but I really think this should be "beta" as it is just too buggy to really use.
